ember-cli-build.js

```diff
@@ -7,6 +7,14 @@ const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 module.exports = function(defaults) {
   let app = new EmberAddon(defaults, {
     // Add options here
+    'ember-hbs-minifier': {
+      whiteList: {
+        elementNodes: ['pre', 'address'],
+        elementClassNames: ['description'],
+        components: ['foo-bar'],
+        partials: []
+      }
```

Comment:
- I'm not sure about the naming of `whiteList`, because the filters essentially blacklist us from doing our transforms. `blackList` sounds confusing too though. maybe just `skip`?

- `elementNodes` -> `elements`

- `elementClassNames` -> `classes`

Reply:
`skip` sounds good. I have used `elementClassNames` to differentiate from `component classNames` since we cannot minify component boundaries(As `classNames and classNameBindings` defined in the js file will be resolved only at runtime).